website/docs/r/policy_set.html.markdown

```diff
@@ -40,10 +60,33 @@ The following arguments are supported:
   all workspaces. Defaults to `false`. This value _must not_ be provided if
   `workspace_external_ids` are provided.
 * `organization` - (Required) Name of the organization.
-* `policy_ids` - (Required) A list of Sentinel policy IDs.
 * `workspace_external_ids` - (Optional) A list of workspace external IDs. If
   the policy set is `global`, this value _must not_ be provided.
+* `vcs_repo` - The [VCS repository settings](#vcs-repository-settings) for this
+  policy set. Forces a new resource if changed.
+* `policies_path` - (Optional) The sub-path within the attached VCS repository
+  to ingress when using `vcs_repo`. All files and directories outside of this
+  sub-path will be ignored. This option can only be supplied when `vcs_repo` is
+  present. Forces a new resource if changed.
+* `policy_ids` - A list of Sentinel policy IDs.
```

Comment:
Something about ordering 😶

Reply:
Ordering? Do you mean alphabetical ordering? Not too sure what you're getting at here.